api/db/schema.sql

```diff
@@ -1,61 +1,79 @@
--- CREATE TABLE users (
---   id SERIAL PRIMARY KEY NOT NULL,
---   profile_photo TEXT,
---   first_name VARCHAR(50),
---   last_name VARCHAR(50),
---   username VARCHAR(50),
---   email VARCHAR(50),
---   password_hash TEXT,
---   latitude TEXT,
---   longitue TEXT,
--- );
-
 CREATE TABLE users (
   id SERIAL PRIMARY KEY NOT NULL,
-  profile_photo TEXT,
   first_name VARCHAR(50),
   last_name VARCHAR(50),
-  username VARCHAR(50),
   email VARCHAR(50),
+  username VARCHAR(50),
+  profile_photo TEXT,
   password_hash TEXT,
   latitude TEXT,
-  longitue TEXT,
-  created_at TIMESTAMP,
-  updated_at TIMESTAMP
+  longitude TEXT,
+  created_at TIMESTAMP DEFAULT now()
 );
 
 CREATE TABLE restaurants (
```

Comment:
Going to need a column for categories (breakfast lunch and dinner)

Reply:
I'll handle this in a separate PR.